sklearn/neighbors/setup.py

```diff
@@ -26,6 +26,12 @@ def configuration(parent_package='', top_path=None):
                          language="c++",
                          libraries=libraries)
 
+    config.add_extension('_sorts',
+                         sources=['_sorts.pyx'],
```

Comment:
I would just use the singular for this module `_sort.py` or alternatively `_sorting.pyx`.

Reply:
Probably `_sorting` is better for consistency (we have `*._testing` for instance).